[Feat/MyGathering] MyMoimView 구현

junil1030 opened the pull request:
## 개요
MyMoimView 구현

## 관련 이슈
- #7 

## 작업 내용
- 내 모임 셀에 보여줄 구체적 내용이 안정해져서 기본적으로 필요할 것 같은 것들로만 구성했습니다.
- 셀의 섬네일은 RoundedRectangle 대체했습니다.
- 뷰 구조 확인부탁드립니다.
<img width="232" height="327" alt="스크린샷 2025-11-07 오후 3 55 57" src="https://github.com/user-attachments/assets/7c832e8a-13fe-47bb-a495-1b8f722a754b" />


### 1115  변경 사항
- ScrollView -> List로 변경하였습니다.
- List가 횡 스크롤은 지원을 하지않아서 LazyHStack으로 변경하였습니다.

## 체크리스트
- [x] 빌드 성공
- [ ] 테스트 완료
- [ ] 에러 처리 완료

junil1030 committed:
feat: ButtonWrapper Modifier 추가
- View extension에 함수로 추가

junil1030 committed:
feat: MyMoimView에서 사용할 Model 추가
- extension으로 임시로 사용할 더미 데이터 생성

junil1030 committed:
feat: 모임 셀 생성

junil1030 committed:
feat: MyMoimViewModel 추가
- 추천 모임과 현재 가입된 모임 프로퍼티 추가
- 임시 데이터를 위한 더미 데이터 로딩 함수 생성

junil1030 committed:
feat: 모임 뷰 생성

nolanMinsung commented on FlatBread/Features/Moim/Presentation/View/MyMoimView.swift:
```diff
@@ -0,0 +1,105 @@
+                LazyHStack(spacing: 12) {
+                    ForEach(moims) { moim in
+                        MyMoimCell(moim: moim)
+                            .frame(width: 380, height: 80)
```
cell의 가로 너비가 숫자로 고정되면 기기 크기가 변했을 때 셀이 잘리거나 공간이 남아보일 수도 있을 것 같습니다.
셀의 너비를 동적으로 구현할 수도 있을 것 같은데, 저도 SwiftUI를 아직 잘 몰라서 같이 공부하면서 개선해나가면 좋을 것 같아요!

nolanMinsung reviewed (approved):
수고하셨습니다!

hyeonghwan commented:
뷰 구조는 이렇게 고정하면 될 것 같습니다
고생하셨습니다

junil1030 committed:
Merge branch 'main' into feat/MyGathering

nolanMinsung committed:
feat: LoginView 파일 추가

nolanMinsung committed:
chore: Sign In with Apple Entitlement 추가

nolanMinsung committed:
feat: SocialLoginError 추가

nolanMinsung committed:
feat: LoginViewModel 추가

nolanMinsung committed:
feat: LoginView 및 초기화면 구현

nolanMinsung committed:
chore: 필요 없는 코드 삭제

nolanMinsung committed:
chore: 필요 없는 코드 삭제, 오타 수정

nolanMinsung committed:
feat: 지도 뷰 카테고리 버튼 컴포넌트 구현

nolanMinsung committed:
feat: 지도 뷰에 카테고리 버튼 목록 추가

더미데이터

junil1030 committed:
Merge branch 'develop' into feat/MyGathering

junil1030 committed:
chore: View 전용 모델로 네이밍 변경

junil1030 committed:
chore: View 전용 모델로 네이밍 변경

junil1030 committed:
refactor: dummy 모델에 imageURL 추가하여 image 불러오도록 개선

nolanMinsung committed:
feat: 탭바 기본 틀 구현

nolanMinsung committed:
feat: 탭바 배경색 설정

nolanMinsung committed:
chore: FBAPIError로부터 SocailLoginError 분리

junil1030 committed:
refactor: MyMoimView List 구조 개선 및 제스처 충돌 해결

  - 두 개의 독립적인 List로 분리 (요즘 뜨는 모임, 가입한 모임)
  - 셀 너비를 동적으로 구현하여 양쪽 여백 추가
  - 좋아요 버튼과 셀 탭 제스처 분리하여 충돌 방지

nolanMinsung committed:
Merge pull request #20 from TheGreatFlatBread/feat/map-category-list

[Feature] 지도 뷰 카테고리 버튼 구현, 탭바 구현

nolanMinsung committed:
chore: main 브랜치 변경사항 반영 및 충돌 해결

nolanMinsung committed:
Merge pull request #16 from TheGreatFlatBread/feat/login

[Feature/Login] 애플 소셜 로그인 구현

junil1030 committed:
Merge branch 'develop' into feat/MyGathering

junil1030 committed:
feat: TabView에 MyMoimView 추가
